From db46fe7ff4deea0fc36557a865ab342f8570fc36 Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard10@gmail.com>
Date: Tue, 26 Aug 2014 11:09:05 -0700
Subject: [PATCH] mkfs: don't enable extended refs by default

We automatically turn on extended refs as needed in kernel.
---
 mkfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkfs.c b/mkfs.c
index 9de61e1..450de41 100644
--- a/mkfs.c
+++ b/mkfs.c
@@ -46,7 +46,7 @@
 
 static u64 index_cnt = 2;
 
-#define DEFAULT_MKFS_FEATURES	(BTRFS_FEATURE_INCOMPAT_EXTENDED_IREF)
+#define DEFAULT_MKFS_FEATURES	(0)
 
 #define DEFAULT_MKFS_LEAF_SIZE 16384
 
-- 
1.7.10.4

